docs/source/common_api_flows.rst

```diff
@@ -2,66 +2,66 @@ Common API workflows
   Choose one datasetId from the result.
-  
+
   Note: Currently, this call only returns public datasets! It is not able to return any private datasets.
   For now, you may need to ask a user for a datasetId or readGroupSetId directly
 
 * ``POST /readgroupsets/search {datasetIds: [<datasetId>]}``
 
   Search for read group sets in a particular dataset. Choose one readGroupSetId from the result.
-  
-  Note: This is a good place to use a `partial request <https://cloud.google.com/genomics/performance#partial>`_ 
-  to only ask for the id and name fields on a read group set. Then you can follow up with a 
+
+  Note: This is a good place to use a `partial request <https://cloud.google.com/genomics/performance#partial>`_
+  to only ask for the id and name fields on a read group set. Then you can follow up with a
   ``GET /readgroupsets/<readGroupSetId>`` call to get the complete read group set data.
-  
+
 * ``GET /readgroupsets/<readGroupSetId>/coveragebuckets``
-  
-  Get coverage information for a particular readset. This will tell you where the read data is located,
+
+  Get coverage information for a particular read group set. This will tell you where the read data is located,
   and which referenceNames should be used in the next step.
-  
-* ``POST /reads/search {readGroupSetIds: [<readGroupSetId>]}``   
 
-  Get reads for a particular read group set. 
-  
-  Note: The call also requires ``referenceName``, ``start`` and ``end``. 
+* ``POST /reads/search {readGroupSetIds: [<readGroupSetId>]}``
+
+  Get reads for a particular read group set.
+
+  Note: The call also requires ``referenceName``, ``start`` and ``end``.
   The referenceName can be chosen from the coverage buckets by the user, along with the
   start and end coordinates they wish to view. The API uses 0-based coordinates.
 
 
-Map reducing over read data within a readset
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Map reducing over read data within a read group set
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 * ``GET /readgroupsets/<readGroupSetId>/coveragebuckets``
 
   First get coverage information for the read group set you are working with.
-  
-  Iterate over the ``coverageBuckets`` array. 
-  For each bucket, there is a field ``range.end``. Using this field, and the number of shards 
-  you wish to have, you can calculate sharding bounds. 
 
-  Let's say there are 23 references, and you want 115 shards. The easiest math would 
-  have us creating 5 shards per reference, each with a ``start`` of ``i * range.end/5`` 
+  Iterate over the ``coverageBuckets`` array.
+  For each bucket, there is a field ``range.end``. Using this field, and the number of shards
+  you wish to have, you can calculate sharding bounds.
+
+  Let's say there are 23 references, and you want 115 shards. The easiest math would
+  have us creating 5 shards per reference, each with a ``start`` of ``i * range.end/5``
   and an ``end`` of ``min(range.end, start + range.end/5)``
```

Comment:
Looking purely at the math for `end`, it will always be `(i+1)/5 * range.end` based on this simplification:

`start` -> `i * range.end/5`

`end`   -> `min(range.end, start + range.end/5)`

`end`   -> `min(range.end, (i * range.end/5) + range.end/5)`

`end`   -> `min(range.end, (i+1) * range.end/5)`

`end`   -> `min(range.end, (i+1)/5 * range.end)`

Since `(i+1)/5 * range.end` <= `range.end` it can be simplified to `(i+1)/5 * range.end`.  If I misinterpreted then maybe more description with an example would help clarify the documentation, so that users are properly implementing it.